examples/r-api/runapi.R

```diff
@@ -0,0 +1,3 @@
+library(dplyr)
```

Comment:
Rename this file `run_api.R`

Reply:
could not make this change since validation check is not allowing '_" special character. So I have renamed it run-api.R